arm64-assembly-optimization.md

```diff
@@ -24,6 +24,34 @@ Some techniques for writing optimized assembly:
 
 We will be adding more sections to this document soon, so check back!
 
+### Other resources
+
+This guide aims only to provide tips on optimizing assembly or intrinsics code.
+There are other resources available on the web which can help you to learn
+concepts or refer to as references. Here a list of helpful pages.
```

Comment:
I think this is a little off: "There are other resources available on the web which can help you to learn
concepts or refer to as references."

There are other resources available on the web which can
- help you to learn concepts
- refer to as references.

There are other resources available on the web which can help you to
- learn concepts
- refer to as references

The "refer to as references" branch of both parsings don't come out right
- "There are other resources available on the web which can refer to as references"
- "There are other resources available on the web which can help you to refer to as references"

I hope that makes sense

Reply:
Thanks for commenting on that. I made a change to the wording. Does it look better to you?

Reply:
I'd say it's still a little awkward, but it does parse now.